googletrans/gtoken.py

```diff
@@ -38,7 +38,8 @@ class TokenAcquirer:
     RE_TKK = re.compile(r'tkk:\'(.+?)\'', re.DOTALL)
     RE_RAWTKK = re.compile(r'tkk:\'(.+?)\'', re.DOTALL)
 
-    def __init__(self, client: httpx.Client, tkk='0', host='translate.google.com'):
+    def __init__(self, client: httpx.Client, tkk='0', host='translate.google.com', verify= True):
+        client.verify = verify;
```

Comment:
This doesn't look good because the client is passed as an argument.

Reply:
Can confirm Line 42 needs to be changed.

Reply:
If this module is going to be refactored to send client options that could accept verifySSL true/false as input argument to client. Then I can close this PR.